console-subscriber/src/lib.rs

```diff
@@ -94,6 +99,12 @@ pub struct ConsoleLayer {
     no_dispatch: Dispatch,
 }
 
+/// A [`Server`] is in charge of aggregating the abstract runtime events into a
+/// model of the runtime's history (such as a list of the currently active
+/// tasks, or statistics summarizing polling times). A [`Server`] also
+/// interprets commands from a client application, such a request to focus in on
+/// a specific task, and translates that into a stream of details specific to
+/// that task.
```

Comment:
technically the server doesn't perform the aggregation, this is done in a separate task that both the layer and the server talk to. i would prefer to focus on what this does in the API; how about something like this?
```suggestion
/// A gRPC [`Server`] that implements the [`tokio-console` wire format][wire].
///
/// Client applications, such as the [`tokio-console CLI][cli] connect to the gRPC
/// server, and stream data about the runtime's history (such as a list of the 
/// currently active tasks, or statistics summarizing polling times). A [`Server`] also
/// interprets commands from a client application, such a request to focus in on
/// a specific task, and translates that into a stream of details specific to
/// that task.
///
/// [wire]: https://docs.rs/console-api
/// [cli]: https://crates.io/crates/tokio-console
```